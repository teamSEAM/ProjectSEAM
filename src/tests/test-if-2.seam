#TEST NESTED IFS#i
entity World:
	func main():
		if(1==1):
			if(2==2):
				screen.out("YES NESTED")
			else:
				screen.out("Nested Failed")
		else:
			screen.out("Nested Failed")
		if(1==1):
			if(2==2):
				screen.out("YES NESTED for second test")
				screen.out("Second Statement Pass")
			else:
				screen.out("Nested Failed")
		else:
			screen.out("Nested Failed")
		if(1==1):
			if(2==1):
				screen.out("2==1 NESTED FAILED")
			else:
				screen.out("Else Nested works")
		else:
			screen.out("Nested Failed")
