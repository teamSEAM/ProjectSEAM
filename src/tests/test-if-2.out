YES NESTED
YES NESTED for second test
Second Statement Pass
Else Nested works
